New ccloud getting started topic (#13323)
cockroachdb/docs: cockroachcloud/cloud-api.md

> The Cloud API is a [REST interface](https://en.wikipedia.org/wiki/Representational_state_transfer) that allows users programmatic access to manage the lifecycle of clusters within their organization.

Reviewer: ```suggestion
The Cloud API is a [REST interface](https://en.wikipedia.org/wiki/Representational_state_transfer) that allows you programmatic access to manage the lifecycle of clusters within your organization.
```
